libpoolprotocols/PoolURI.h

```diff
@@ -9,49 +10,26 @@ namespace dev
 enum class SecureLevel {NONE = 0, TLS12, TLS, ALLOW_SELFSIGNED};
 enum class ProtocolFamily {GETWORK = 0, STRATUM};
 
-class URI
+class URI : network::uri
```

Comment:
Don't do that :)

https://github.com/cpp-netlib/uri/wiki/Extending-the-URI

preferred way would be variant 3

Reply:
Also do we really need the URI class for that? As mentioned in chat earlier, i think ``m_uri`` would be better in ``PoolConnection``